docs/specification/current.md

```diff
@@ -639,6 +640,81 @@ Field | Optionality | Type | Description
+
+Upon the user accepting a suggestion (perhaps when she clicks a displayed label (e.g., button) from a "suggestion" card), the CDS client informs the service by posting the card and suggestion `uuid`s to the CDS Service's feedback endpoint with an outcome of `accepted`.
+
+To enable a positive clinical experience, the analytics endpoint may be called for multiple hook instances or multiple cards at the same time or even multiple times for a card or suggestion. Depending upon the UI and workflow of the CDS client, a CDS Service may receive feedback for the same card instance multiple times. 
+
+```
+POST {baseUrl}/cds-services/{serviceId}/feedback
+
+{
+   "feedback":[
+      {
+         "card":"`card.uuid` from CDS Hooks response",
+         "outcome":"accepted",
+         "acceptedSuggestions": [ { "id" : "`card.suggestion.uuid` from CDS Hooks response" } ],
+         "outcomeTimestamp": "iso timestamp in UTC when action was taken on card"
+      }
+   ]
+}
+```
+
+If either the card or the suggestion has no `uuid`, the CDS client does not send a notification.
+
+## Card ignored
+
+If the end-user doesn't interact with the CDS Service's card at all, the card is *ignored*. In this case, the CDS Client does not inform the CDS Service of the rejected guidance. Even with a `card.uuid`, a `suggestion.uuid`, and an available feedback service, the service is not informed. 
+
+## Overridden guidance
+
+A CDS client may enable the end user to override guidance without providing an explicit reason for doing so. The CDS client can inform the service when a suggestion was dismissed by specifying an outcome of `overridden` without providing an `overrideReason`. This may occur, for example, when the end user viewed the card and dismissed it without providing a reason why.
+
+```
+POST {baseUrl}/cds-services/{serviceId}/feedback
+
+{
+   "feedback":[
+      {
+         "card":"f6b95768-b1c8-40dc-8385-bf3504b82ffb", // uuid from `card.uuid`
+         "outcome":"overridden",
+         "outcomeTimestamp": "iso timestamp in UTC when action was taken on card"
+      }
+   ]
+}
+
+```
+
+## Explicit reject with override reasons
+
+A CDS client can inform the service when a suggestion was rejected by POSTing an outcome of `overridden` along with an `overrideReason` to the service's feedback endpoint. The CDS Client may enable the clinician to supplement the `overrideReason` with a free text comment, supplied to the CDS Service in `overrideReason.userComment`. 
+
+```
+POST {baseUrl}/cds-services/{serviceId}/feedback
+
+{
+   "feedback":[
+      {
+         "card":"9368d37b-283f-44a0-93ea-547cebab93ed",
+         "outcome":"overridden",
+         "overrideReason": { "key" : "reason-id-provided-by-service", "userComment" : "clinician entered comment>" }
```

Comment:
'key' is left over from an earlier version of the overrideReasons PR.  Now we need the system+code